src/index.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process')
+
+module.exports = function (pluginConfig, config, cb) {
+  exec('git tag', (err, stdout, stderr) => {
+    if (err) return cb(err)
+
+    const versionRegex = /^v((?:[0-9]+\.){2}[0-9]+)$/
+
+    const tags = stdout
+                  .trim()
+                  .split('\n')
+                  .map((t) => t.trim())
+                  .filter(t => versionRegex.test(t))
+
+    if (tags.length < 1) return cb(null, {})
+
+    const tag = tags.pop()
+    const version = tag.match(versionRegex)[1]
+
+    cb(null, {
+      version: version,
+      gitHead: tag
```

Comment:
I like that you're just using the tag name for this. Thinking of any downsides, but I think I can't find any.